.github/workflows/release.yaml

```diff
@@ -0,0 +1,243 @@
+name: Publish release
+
+env:
+  GRADLE_OPTS: '-Dorg.gradle.daemon=false -Dkotlin.compiler.execution.strategy=in-process'
+  JAVA_VERSION: 11
+  NODEJS_VERSION: 14
+  PROTOBUF_VERSION: 3.10.1
+  BINTRAY_API_USER: ${{ secrets.BINTRAY_API_USER }}
```

Comment:
These need to be configured as Github actions secrets and then the Bintray publishing extension should pick these up while running the Gradle publishToBintrayRepository task.

Based on the environment variable names here: https://github.com/streem/pbandk/blob/master/buildSrc/src/main/kotlin/addBintrayRepository.kt

Reply:
I've added them to GitHub secrets